modules/core/src/main/java/org/apache/ignite/logger/java/JavaLogger.java

```diff
@@ -214,13 +205,26 @@ public JavaLogger(final Logger impl, boolean configure) {
         quiet = quiet0;
     }
 
+    /**
+     * Creates new logger with given parameters.
+     *
+     * @param impl Java Logging implementation to use.
+     * @param quiet Quiet mode.
+     * @param cfg Path to configuration.
+     */
+    private JavaLogger(Logger impl, boolean quiet, String cfg) {
+        this.impl = impl;
+        this.quiet = quiet;
+        this.cfg = cfg;
+    }
+
     /** {@inheritDoc} */
     @Override public IgniteLogger getLogger(Object ctgr) {
         return new JavaLogger(ctgr == null
             ? Logger.getLogger("")
             : Logger.getLogger(ctgr instanceof Class
                 ? ((Class<?>)ctgr).getName()
-                : String.valueOf(ctgr)));
+                : String.valueOf(ctgr)), quiet, cfg);
```

Comment:
It starts invoking another constructor. Before your change there was a chain:

```JavaLogger#getLogger(Object) -> new JavaLogger(Logger, configure=true)```

After your change, in case root logger created as `new JavaLogger(false)` the underlying logger will use constructor with no configuration and it never been configured. And the constructor `JavaLogger(Logger, boolean)` is useless. Is it ok?